MapboxMobileEvents/MapboxMobileEvents.h

```diff
@@ -1,10 +1,10 @@
 #import <UIKit/UIKit.h>
 
 //! Project version number for MapboxMobileEvents.
-FOUNDATION_EXPORT double MapboxMobileMetricsVersionNumber;
+FOUNDATION_EXPORT double MapboxMobileEventsVersionNumber;
 
 //! Project version string for MapboxMobileEvents
-FOUNDATION_EXPORT const unsigned char MapboxMobileMetricsVersionString[];
+FOUNDATION_EXPORT const unsigned char MapboxMobileEventsVersionString[];
```

Comment:
Technically this is a backwards-incompatible change, since clients have access to this constant and can use it for additional safety with conditional linking. Would it be possible to retain the old names as deprecated aliases for the new ones?